src/js/TuningSliders.js

```diff
@@ -62,12 +62,31 @@ TuningSliders.initialize = function() {
     this.updateFilterSlidersDisplay();
 };
 
-TuningSliders.setExpertMode = function(expertMode) {
-    this.expertMode = expertMode;
-    if (this.expertMode) {
+TuningSliders.setExpertMode = function() {
+    this.expertMode = isExpertModeEnabled();
+    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+        if (this.expertMode) {
+            if ($('.pid_tuning input[name="dMinRoll"]').val() == 0) {
+                $('.tab-pid_tuning .DMinRatioSlider').hide();
+            } else {
+                $('.tab-pid_tuning .DMinRatioSlider').show();
+            }
+            $('.tab-pid_tuning .MasterSlider').show();
+            $('.tab-pid_tuning .advancedSlider').show();
+        } else {
+            $('.tab-pid_tuning .MasterSlider').hide();
+            $('.tab-pid_tuning .advancedSlider').hide();
+            $('.tab-pid_tuning .DMinRatioSlider').hide();
+        }
```

Comment:
This `if` can be totally simplified using the `toggle()` function in place of `show()` and `hide()`, as I suggested in my first review:
```javascript
const dMinShow = $('.pid_tuning input[name="dMinRoll"]').val() !== 0;
$('.tab-pid_tuning .DMinRatioSlider').toggle(this.expertMode && dMinShow);
$('.tab-pid_tuning .advancedSlider').toggle(this.expertMode);
$('.tab-pid_tuning .MasterSlider').toggle(this.expertMode);
```
And seeing that the `MasterSlider` shows and hides exactly the same than the `advancedSlider` is better simply to add the `advancedSlider` class to the `MasterSlider` and remove this toggle too.

Reply:
@McGiverGim , I get weird behavior with this method for dMinShow (see below).  :-/

MasterSlider needs to be its own class so we can call it to show for API < 1.44.

![M6if0AES8s](https://user-images.githubusercontent.com/25570978/108379304-cad61d00-71d3-11eb-82b4-f77aa65781d4.gif)

Reply:
Of course, your Master Slider can be shown it hide with other conditions in other places.
But if you get word behavior something is wrong with the code, toggle is exactly the same than show/hide with a if.

Reply:
```
TuningSliders.setExpertMode = function() {
    this.expertMode = isExpertModeEnabled();
    const dMinShow = $('.pid_tuning input[name="dMinRoll"]').val() !== 0;
    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
        $('.tab-pid_tuning .DMinRatioSlider').toggle(this.expertMode && dMinShow);
        $('.tab-pid_tuning .advancedSlider').toggle(this.expertMode);
        $('.tab-pid_tuning .MasterSlider').toggle(this.expertMode);
        $('#slidersPidsBox, #slidersFilterBox').removeClass('nonExpertModeSliders');
    } else {
        if (this.expertMode) {
            $('#slidersPidsBox, #slidersFilterBox').removeClass('nonExpertModeSliders');
        } else {
            $('#slidersPidsBox, #slidersFilterBox').addClass('nonExpertModeSliders');
        }
        $('.tab-pid_tuning .advancedSlider').hide();
        $('.tab-pid_tuning .DMinRatioSlider').hide();
    }
};
```

Reply:
@McGiverGim @spatzengr reverted this to your version and now it's working after making the required changes.

Reply:
It needs to be const` dMinShow = parseInt($('.pid_tuning input[name="dMinRoll"]').val()) !== 0;` to work.  It works now fine.